### DISCUSSION TITLE
2019/02/access-point-ap-dengan-modem-zte-f609

### DISCUSSION THREAD

**Discussion by tegursapa** [madi10/MANTANKODE#15] [General]
# 2019/02/access-point-ap-dengan-modem-zte-f609

cara merubah modem zte f609 bekas indihome sebagai AP (Access Point) Mikrotik

https://www.mantankode.net/2019/02/access-point-ap-dengan-modem-zte-f609.html

**Comment by madi10** [comment]
Up, keren hehe.
